packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionsTableRowAction.tsx

```diff
@@ -28,7 +33,10 @@ const GetHelpButton = ({
     <Button
       variant={variant}
       onClick={onClick}
-      className={variant === 'secondary' ? 'bg-white px-4 py-3' : ''}
+      className={twMerge(
+        'w-16 rounded',
+        variant === 'secondary' ? 'bg-white px-4 py-3' : ''
```

Comment:
i think you can use && rather than ternary
if it shows `false`, just wrap it with `twJoin`

Reply:
updated